CHANGELOG.md

```diff
@@ -0,0 +1,105 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+ 
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+
+
```

Comment:
Misschien ook meteen de huidige versie toevoegen? :) 

Dan is duidelijk dat vanaf `0.11.16` deze Changelog wordt bijgehouden.